### ISSUE TITLE
Can't close the windows

### ISSUE THREAD

**Issue by Kcchouette** [chapeaulinux/hht#1]
Hello

I can't close the windows using the cross at the right of the software. I need to use the exit button.

Is it normal or not?

**Comment by vpooley** [comment]
Yes this is intentional, use the back or exit buttons in the dialogues.

**Comment by Kcchouette** [comment]
So it would be kind to remove the cross (that doesn't work).
